action/blob_data.go

```diff
@@ -166,6 +166,11 @@ func (tx *BlobTxData) ValidateSidecar() error {
 	if permitted := params.MaxBlobGasPerBlock / params.BlobTxBlobGasPerBlob; size > permitted {
 		return errors.Errorf("too many blobs in transaction: have %d, permitted %d", size, params.MaxBlobGasPerBlock/params.BlobTxBlobGasPerBlob)
 	}
```

Comment:
can move to SanityCheck

Reply:
They could be moved into `verifySidecar`